andrewinsoul/MyDiary
# Ft user login #159382100

### 111504508 Build out REST Endpoints for Resources (CRUD) 

What does this PR do?
- This PR was created for review purposes on the user login functionality

Description of Task to be completed
- Log in a user and return token if supplied credentials are correct and send appropriate error message if log in operation was unsuccessful

How should this be manually tested?
- Ensure server is running, access the login endpoint (/auth/login) with correct credentials, a token will be returned

What are the relevant pivotal tracker stories?
- <a href='https://www.pivotaltracker.com/story/show/159382100'>Link to Story</a>
